src/GUI/CafeBill.java

```diff
@@ -131,7 +136,7 @@
 	public float db_tax3 =(float) 0.0;
 	public float db_totalTaxPerc = (float)0.0;
 
-	private java.sql.Time getCurrentTime() {
+	public static java.sql.Time getCurrentTime() {
```

Comment:
Instead of making this public static create a helper class that will have public static methods